tests/test_adjoint.py

```diff
@@ -49,15 +70,18 @@ def test_adjoint_F(self, mkey, shape, kernel, space_order):
                         coordinates=solver.geometry.src_positions)
 
         # Run forward and adjoint operators
-        rec, _, _ = solver.forward(save=False)
+        if setup_func.__name__ == 'acoustic_setup':
+            rec, _, _ = solver.forward(save=False)
+        else:
+            rec, _, _, _ = solver.forward(save=False)
```

Comment:
can do `rec = solver.forward(save=False)[0]` as a one line and avoid if/else

Reply:
Done